bpf/lib/common.h

```diff
@@ -546,8 +546,7 @@ struct lb6_service_v2 {
 	__u32 backend_id;
 	__u16 count;
 	__u16 rev_nat_index;
-	__u16 weight;
-	__u16 pad;
+	__u32 pad;
```

Comment:
Any concerns about struct alignments here on upgrade? This isn't my area of expertise, so I may be worrying for no reason.

Reply:
+1

Reply:
Fair point, but it doesn't change. Otherwise, the alignchecker would report the error.